### DISCUSSION TITLE
Using dataview as data source return error: cannot read properties of undefined (reading "path")

### DISCUSSION THREAD

**Discussion by ycwu1030** [marcusolsson/obsidian-projects#603] [Help]
Hi:
   New to Projects. When using folder as data source, there is no problem. However, when I use the dataview as source, it returns the TypeError: Cannot read properties of undefined (reading 'path'). 

It is a quite simple dataview query:

```
TABLE alias as "ALIAS", Rating as "RATES" FROM #Resource and "03.Resources"
```

Is there anything missed? Thanks!

Best
ycwu

**Comment by mlevison** [comment]
I have the same problem with a different Query: 
TABLE join(rows.Author, "<br>") AS Author, join(rows.file.link, "<br>") AS Book
FROM "Sources/BookNotes"
WHERE contains(status, "Unread")
WHERE Category != "ADHD" AND Category != "Biography & Autobiography" AND Category != "Food" AND Category != "Fiction" AND Category != "Friendship" AND Category != "Health & Fitness" AND Category != "History" AND Category != "Photography" AND Category != "Relationships" AND Category != "Self-Help" AND Category != "Travel"
GROUP BY Category

**Comment by mlevison** [comment]
In my case the GROUP BY Category was the killer.
